libs/zjobs/src/zjobs.zig

```diff
@@ -577,8 +563,15 @@ pub fn JobQueue(
             const prereq_size = @sizeOf(JobId);
             const max_prereqs = (max_job_size - jobs_size) / prereq_size;
 
-            jobs: *Self = .{},
-            prereqs: [max_prereqs]JobId = [_]JobId{JobId.none} ** max_prereqs,
+            jobs: *Self,
+            prereqs: [max_prereqs]JobId,
```

Comment:
I don't see any reason to move the initialization of `prereqs` into `init()`.  Please explain.